Formula/qdae.rb

```diff
@@ -20,10 +20,12 @@ class Qdae < Formula
     sha256 x86_64_linux:  "663f3822c76388f597539c10e9296fb58289d364902b86497f037843cce25d85"
   end
 
-  depends_on "libxml2"
   depends_on "sdl"
 
+  uses_from_macos "libxml2"
```

Comment:
Were the existing bottles built with linkage to Homebrew's `libxml2`?

If so, we should probably revision bump if we switch to system `libxml2` and remove `CI-no-bottles` to make sure dependencies align with those needed by bottles.